### PR TITLE
Pull images without printing progress information

### PR DESCRIPTION
**Pull request by szepeviktor**
Spare ~1000 lines of CLI graphics.
https://github.com/johnbillion/query-monitor/actions/runs/5788135287/job/15686995953#step:11:14

📚 https://docs.docker.com/engine/reference/commandline/compose_up/#options

### CODE REVIEW

File: .github/workflows/reusable-acceptance-tests.yml [hunk 1]
```diff
@@ -79,13 +79,13 @@ jobs:
...
     - name: Start Docker environment
-      run: composer test:start
+      run: composer run test:start -- --quiet-pull
 
     - name: Log WordPress version
       run: docker compose run --rm wpcli core version
 
     - name: Run the tests
-      run: composer test:acceptance
+      run: composer run test:acceptance
```
Comment (szepeviktor) [inline .github/workflows/reusable-acceptance-tests.yml:88]: `composer test:acceptance` says `test:acceptance` is a Composer command.
`composer run test:acceptance` says `test:acceptance` is a Composer script. Which it is.

**Comment by szepeviktor** [comment]
> `run: composer test:start`

BTW I would not make this Composer script call dependent on projects using this repository.
Would just say `docker compose up ...` here.

**Comment by johnbillion** [comment]
Thanks 👍

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**